Docs/source/usage/parameters.rst

```diff
@@ -320,6 +320,12 @@ Embedded Boundary Conditions
     the electric potential at the surface of the embedded boundary,
     as a function of  `x`, `y`, `z` and time.
 
+    .. warning::
```

Comment:
I think adding this as regular two sentences is ok for this section. It's brief and part of the definition.